blocks/core/ff_module/ff_module-task/_ff_module-task-renderer.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var React = require('react'),
+    ReactDOM = require('react-dom');
+
+
+var Task = require('./ff_module-task.js'),
+    selector = 'data-ff_module-task';
+
+var props = {
+    modifier: 'wibble',
+    classes: 'another-class',
+    from: 'Big Daddy',
+    to: 'A Class',
+    duedate: "27/08/2018",
+    message: 'A big message from Big Daddy',
```

Comment:
This made me laugh, but our pattern lib is publicly accessible. Can we change this to more meaningful test data?